tpgtools/utils/strings.go

```diff
@@ -33,13 +33,13 @@ var initialisms = map[string]string{
 }
 
 // snakeToTitleCase converts a snake_case string to TitleCase / Go struct case.
```

Comment:
Can you fix the function name in these comments to match the new name of the function?

Reply:
will do